packages/field/src/FormControlMixin.js

```diff
@@ -55,6 +55,11 @@ export const FormControlMixin = dedupeMixin(
            * Contains all elements that should end up in aria-describedby of `._inputNode`
            */
           _ariaDescribedNodes: Array,
+          /**
+           * Based on the role, details of handling model-value-changed repropagation
+           * @type {'child'|'form-group'|'choice-group'}
```

Comment:
form-group? why not fieldset?

Reply:
same with child, why not field?

Reply:
`fieldset` is indeed a better fit 👍 
 `child` because it can either be an option or a field.

Reply:
Hmm, will be a bit weird if I do, because then The formGroupMixin either wouldn't have a role(would inherit 'child' then) or would have the role `fieldset` (and there is a subtle difference between form-group and fieldset, the latter allows children with different names).